code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -194,6 +194,9 @@
 	..()
 
 /datum/reagent/consumable/drink/milk/on_skeleton_life(mob/living/carbon/human/H)
+	if(type in subtypesof(/datum/reagent/consumable/drink/milk))		//prevents stacking subtypes for healing purposes. only pure milk gives strong bones and calcium.
```

Comment:
Not sure why this got marked as outdated since I didn't touch this line in the commit after you commented it...

Effect is the same in your code and mine, except mine avoids a literal in the conditional.